nano/node/messages.cpp

```diff
@@ -940,6 +940,21 @@ std::size_t nano::confirm_req::size (nano::block_type type_a, std::size_t count)
 	return result;
 }
 
+std::string nano::confirm_req::to_string () const
+{
+	std::stringstream stream;
+
+	stream << header.to_string ();
+
+	for (auto roots_hash = roots_hashes.begin (), end = roots_hashes.end (); roots_hash != end; ++roots_hash)
```

Comment:
This line is too complicated. Please use range loop to simplify this code.
https://en.cppreference.com/w/cpp/language/range-for

Reply:
this should be resolved after the recent push